examples/jsm/exporters/GLTFExporter.js

```diff
@@ -53,6 +54,18 @@ class GLTFExporter {
 
 		} );
 
+    this.register( function ( writer ) {
```

Comment:
nit: indentation doesn't look right on some of these lines.

Reply:
Fixed.

Reply:
Thanks! I thought it was too good to be true that the linter passed on my first try...